titan/settings/nyc-msm/location.yml

```diff
@@ -1,55 +1,68 @@
 location:
   scaling:
-    high_test:
-      demographics|white|ppl:
+    high_test_w:
+      demographics|white|ppl: &all_ags
```

Comment:
I'd spell out `ags` to be `agents` I was reading this as "no aggregates" at first